portfolio/src/main/java/com/google/sps/data/TemporaryCommentStore.java

```diff
@@ -0,0 +1,22 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TemporaryCommentStore implements CommentStore {
+
+  private List<Comment> comments = new ArrayList<>();
+
+  @Override
+  public List<String> getComments() {
+    return comments.stream()
+        .map(c -> c.getComment() + "\nPosted by " + c.getName())
```

Comment:
I think \n is part of presentation, not logic, so it belongs to html instead of here. Could this function return list of pairs "comment, name", where "Posted by" is added in the formatting part in JS? WDYT?

Reply:
In task 5 where I started using the datastore, I realised that this isn't necessarily the most flexible format for layout on the front end so I have changed it to return a List of Comment objects. So this has been changed on the next pull request.